modules/migrations/migrate.go

```diff
@@ -127,11 +127,6 @@ func migrateRepository(downloader base.Downloader, uploader base.Uploader, opts
 			if err != nil {
 				return err
 			}
-			for _, issue := range issues {
```

Comment:
Why remove this?

Reply:
Because with these changes we should not have to modify the content of original issue and comment to include "Author: lunny" since we will display it elsewhere.

Reply:
Oh. Right and I think maybe we could change opts.IgnoreIssueAuthor to true outside of function `migrateRepository` but not change this.

Reply:
I like to remove it because I think it is bad practice to change the content of a comment/issue when importing if we don't need to (I know why we did it originally and that made sense to help start migration, but it should be not needed after this).

Is there a case you think we should need this code still?

Reply:
Then we should also remove `opts.IgnoreIssueAuthor`.

Reply:
done